pair-programming/week-4/hello-component/Hello.js

```diff
@@ -1,10 +1,8 @@
-export default function Hi({ firstName, lastName }) {
+export default function Hello({ firstName, lastName }) {
+  // TODO: Render a greeting like "Hello, [firstName] [lastName]"
   return (
-    <div className="Hi">
-      <h1>Hello Component</h1>
-      <h2>
-        Hello, {firstName} {lastName}
-      </h2>
+    <div className="Hello">
+      <h2>Hello, {firstName} {lastName}</h2>
```

Comment:
Let's keep the first header as well for the learning participants that still need to distinguish between the two.

```suggestion
      <h1>Hello Component</h1>
      <h2>Hello, {firstName} {lastName}</h2>
```